apps/meteor/server/models/raw/LivechatRooms.ts

```diff
@@ -2383,13 +2383,13 @@ export class LivechatRoomsRaw extends BaseRaw<IOmnichannelRoom> implements ILive
 		return this.deleteOne(query);
 	}
 
-	setVisitorLastMessageTimestampByRoomId(roomId: string, lastMessageTs: Date) {
+	setVisitorLastMessageTimestampByRoomId(roomId: string, contactLastMessageAt: Date) {
 		const query = {
 			_id: roomId,
 		};
 		const update = {
 			$set: {
-				'v.lastMessageTs': lastMessageTs,
+				contactLastMessageAt,
```

Comment:
what is the point of adding this field? is this really necessary to fix the problem? or its like a improvement/refactor?
I could not find the exact line that actually fixes the problem